detoks/src/main/java/nl/tudelft/trustchain/detoks/Recommender.kt

```diff
@@ -17,23 +17,20 @@ class ProfileEntry(
     }
 }
 
-class Profile(
-    val magnets: HashMap<String, ProfileEntry> = HashMap()
-) {
-    fun updateEntryWatchTime(torrent: TorrentManager.TorrentHandler, time: Long) {
-        val name = torrent.torrentName + "[" + torrent.fileName + "]"
-        if(!magnets.contains(name)) magnets[name] = ProfileEntry()
-        magnets[name]!!.watchTime += time
-        Log.i("DeToks", "Updated watchtime of $name to ${magnets[name]!!.watchTime}")
+class Profile(val magnets: HashMap<String, ProfileEntry> = HashMap()) {
+    fun updateEntryWatchTime(torrent: String, time: Long) {
+        if(!magnets.contains(torrent)) magnets[torrent] = ProfileEntry()
+        magnets[torrent]!!.watchTime += time
+        Log.i("DeToks", "Updated watchtime of $torrent to ${magnets[torrent]!!.watchTime}")
     }
 }
 
 class Recommender {
-    private fun coinTossRecommender(magnets: HashMap<String, ProfileEntry>): Map<String, ProfileEntry> {
+    private fun coinTossRecommender(magnets: HashMap<String, ProfileEntry>) : Map<String, ProfileEntry> {
```

Comment:
very small nitpick but I noticed that you added a space between the double dots and the closing bracket you probably did that by mistake